### ISSUE TITLE
Air Handler no longer reporting temperature

### ISSUE THREAD

**Issue by chrisferagotti** [geoffdavis/esphome-mitsubishiheatpump#159]
First, I'd like to say thank you for this thing. It's been great.

Now, my problem. (I use Fahrenheit instead of Celsius, as an FYI)

I was trying to set up a remote temperature sensor for one of our air handlers, following the directions in the README. After doing it, it was reporting 32 degrees, which is way colder than it actually was. I thought maybe there was a problem with some F <-> C conversion going on, so I just decided to go back to my old config and deal with it later. Well, after going back, now it's not reporting a temperature at all, which means it never turns on. Below is what two of my air handlers look like.

![image](https://github.com/geoffdavis/esphome-mitsubishiheatpump/assets/1155566/0ecbd83c-0f9a-48df-ab02-93e91c42492e)

This is kind of a huge deal, and it seems like I've killed my air handler. I've tried to go back, again, to setting up the remote sensor, but I have the same results. Below is my yaml file (renamed, since it won't let me upload a yaml file.)

[heat-pump--bedroom.txt](https://github.com/user-attachments/files/16168953/heat-pump--bedroom.txt)

**Comment by chrisferagotti** [comment]
Well, if I was a little better at reading comprehension, I'd have picked up that I needed to convert the value to Celsius.

Everything seems to be working now. I'm still a little confused on why, when trying to go back to previous settings, it didn't work (and it gives me some concern for the fallback option to the built-in sensor if needed), but no need to keep tickets open for issues that _might_ come up. Closing!
